modules/ROOT/pages/cloudhub-deployment-settings.adoc

```diff
@@ -0,0 +1,34 @@
+= Configuring CloudHub Deployment Settings
+
+To configure settings for deploying Anypoint Partner Manager to CloudHub, access the business unit and environment for which you want to deploy Partner Manager and then follow these steps:
+
+. In the *Configure a Partner Manager environment* window, enter the following general settings:
++
+[%header%autowidth.spread]
+|===
+| Setting | Description | Default 
+| Deployment Target Type | Target type for your Partner Manager deployment. Select *CloudHub 1.0*. | CloudHub 1.0
+| Runtime Version | Mule runtime engine version to run Partner Manager applications.| 4.4.0
+| CloudHub Region |Region in CloudHub to which you want to deploy Partner Manager runtime applications. You can't change the region if active message flows are deployed to the existing region. To change the region in this case, you must undeploy the message flows. You can redeploy the message flows after you change the region.| Retrieved from the business group's Runtime Manager setting
+| Anypoint MQ Region | This value is automatically assigned based on the specified CloudHub region and you can't change it. |  
+| Automatically restart app when not responding | Directs CloudHub to automatically restart your application when the monitoring system discovers an issue. | Selected
+|===
++
+. Select *Applications* and enter the following application settings:
++
+[%header%autowidth.spread]
+|===
+| Setting |Description | Default
+| Each B2B endpoint | Reserved CPU space, CPU space limit, and memory capacity for each Partner Manager endpoint. Partner Manager deploys a generic Mule app to the runtime for each endpoint protocol and direction (receive or send) combination. a| 0.1 vCores
+| B2B API Replication Service | Reserved CPU space, CPU space limit, and memory capacity for the B2B API Replication Service. | 0.2 vCores
+| B2B Document Processing Service | Reserved CPU space, CPU space limit, and memory capacity for the B2B Document Processing Service. | 0.2 vCores
+| Deploy HTTP/AS2 apps to private port | Directs CloudHub to automatically deploy newly-created AS2, HTTP, and HTTPS endpoints to a private port.| Selected
```

Comment:
Our style is to not hyphenate adverbs (Do not use a hyphen after any word ending in "ly".)